homeassistant/components/wemo/fan.py

```diff
@@ -221,10 +180,17 @@ def turn_off(self, **kwargs) -> None:
 
         self.schedule_update_ha_state()
 
-    def set_speed(self, speed: str) -> None:
+    def set_percentage(self, percentage: int) -> None:
```

Comment:
We have code to guard for `None`. Should this be?

```suggestion
    def set_percentage(self, percentage: Optional[int]) -> None:
```